extensions/sql-database-projects/src/models/connections/connectionService.ts

```diff
@@ -42,7 +42,7 @@ export class ConnectionService {
 				saveProfile: false,
 				id: '',
 				connectionName: profile.profileName,
-				options: [],
+				options: { 'encrypt': 'false' },
```

Comment:
just wondering, is it only encrypt that the value needs to be string or all options? Unless the value needs to be a string for all options that get set like this, it's not intuitive that you need to use `'false'` rather than `false`